src/db.c

```diff
@@ -308,6 +308,57 @@ static const char *sort_clause[] =
     "ORDER BY shuffle_pos ASC",
   };
 
+
+/* Keep in sync with enum media_kind */
+static char *media_kind_label[] = { "music", "movie", "podcast", "audiobook", "musicvideo", "tvshow" };
+
+const char *
+db_media_kind_label(enum media_kind media_kind)
+{
+  int index;
+
+  index = bitmask_to_index(media_kind);
```

Comment:
Have you considered if a switch would be simpler + safer? The below breaks if the enums don't follow the current structure.

Reply:
I wanted to find a better solution than how it is done for the listener events. There we have long if statements to translate the event to a string and a string to the event (websockets.c and mpd.c). 
To add an event to the websocket, there are three places in the file that need to be modified. And maybe the generic translation of enums and labels could make this easier.

But i am indifferent, simple ifs or switch statements will also do. And for the listener events it might not be so easy to apply (there it is truly a bit mask). If you prefer the simple solution, i will change it.

Reply:
What do you think about an array then? With a struct that has the enum and the label? Then a for-loop can translate, and it would be fairly simple to see what is going on.

Reply:
I like it, keeps it simple. It is implemented in 31c0066